salt/cleanup/remove-post-orchestration.sls

```diff
@@ -10,7 +10,7 @@ include:
 # k8s cluster
 ###############
 
-{%- set k8s_nodes = salt['mine.get']('roles:(kube-master|kube-minion)', 'nodename', expr_form='grain_pcre').keys() %}
+{%- set k8s_nodes = salt.caasp_nodes.get_with_expr('G@roles:kube-master', booted=True) %}
```

Comment:
Missed this on previous PR, maybe `bootstrapped` is closer to what we mean here.

Reply:
Yeah, I mean that we could rename it. Anyway, it's a minor thing.

Reply:
Type `bootstrapped` five times in a row... Now you see why I named it `booted` :-D